src/gmc-display-config.c

```diff
@@ -25,6 +25,12 @@
 
 #include <stdio.h>
 
+#if GLIB_CHECK_VERSION(2, 68, 0)
+#else
+#define g_memdup2(ptr,sz) (G_LIKELY(((guint64)(sz)) < G_MAXUINT)) ? g_memdup(ptr,sz) : (g_abort(),NULL)
+#endif
```

Comment:
I'd rather it'd be `#ifdef HAVE_MEMDUP2 .. =  g_memdup2(..); #else .. =  g_memdup(..); #endif`  where it's used.

Reply:
I'm not an expert in C/C++ code, I took that from somewhere :)
Feel free to use better variant.